DigitalLearningSolutions.Web/Scripts/spec/contentViewer.spec.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom

```

Comment:
What's this for? 🙂

Reply:
In our current `testEnvironment` in the Jest config is `node`, and this leads to the `window` global being undefined and the `window.closeMpe = closeMpe` failing. One of the ways of fixing this problem is to change environment, according to [this](https://stackoverflow.com/questions/46274889/jest-test-fails-with-window-is-not-defined), but there may be others. Globally changing the environment breaks the other tests so this does it just for this file 🙂

Reply:
I see, sounds good 🙂